src/Form/Form.js

```diff
@@ -136,8 +136,12 @@ class Form {
         this.handlerExecuted = true
     }
 
-    /** @return {DataStorageObject} */
-    getValues () {
+    /**
+     * Reads the values from the form
+     * @param {boolean} shouldPrepareForStorage
+     * @return {DataStorageObject}
+     */
+    getValues (shouldPrepareForStorage = true) {
```

Comment:
NAB: Worth using an object so we can provide a name for the boolean when we call this?
```suggestion
    getValues ({ shouldPrepareForStorage } = { shouldPrepareForStorage: true }) {
```

Reply:
No longer needed.